site/docs/foundations/data-visualization/chart-accessibility.mdx

```diff
@@ -0,0 +1,88 @@
+
+<Image
+  src="/img/foundations/dataviz-hex.png"
+  srcDark="/img/foundations/dataviz-hex-dark.png"
+  alt="Diagram of best practices using categorical color versus color picking."
+/>
+
+### Color order
+
+Salt's categorical colors are intended to be used in numerical order, from 1 through 20.
+
+<Image
+  src="/img/foundations/dataviz-color-order.png"
+  srcDark="/img/foundations/dataviz-color-order-dark.png"
+  alt="Diagram of the categorical color palette."
+/>
+
+A well-defined, distinct color palette ensures that everyone can easily interpret the data. Using the set categorical palette colors minimizes the risk of misinterpretation and ensures a visual order. Don’t apply similar categorical color colors to the same chart.
+
+<Image
+  src="/img/foundations/dataviz-monochromatic.png"
+  srcDark="/img/foundations/dataviz-monochromatic-dark.png"
+  alt="Diagram of best practices using categorical palette versus monochromatic palette."
+/>
+
+You can enhance your chart’s accessibility by adopting additional visual strategies other than color such as:
+
+- Pattern overlays for different categories
+- Direct labelling where possible
+- Adding a border with at least a 3:1 contrast ratio against the background to distinguish between segments
+
+You can present chart data in a nearby table or list, in an overlay, or -if necessary- as a last resort on a separate screen to support screen readers. However, placing essential information on a separate screen can disrupt the flow of information. Screen reader users may need to navigate back and forth between different sections to access the full text alternative for a complex image, which can make understanding the data more difficult.
+
+For more information about text alternatives, read the [JPMC UX Accessibility Resources](https://go/text-alternatives#TextAlternatives-Compleximages) (Internal users only).
+
+## Line patterns
+
+Color shouldn't be the only visual cue used in charts. Always include additional cues such as text, patterns, or symbols to help users distinguish between different data sets. For example, use line patterns to differentiate multiple line series. Patterns and other visual indicators improve readability, especially for users with color vision requirements.
+
+<Image
+  src="/img/foundations/dataviz-line-patterns2.png"
+  srcDark="/img/foundations/dataviz-line-patterns2-dark.png"
+  alt="Diagram of line chart using line patterns."
+/>
+
+## Pattern fills
+
+Use pattern fills for two-dimensional shapes, such as bars and slices. Pattern fills should always be black and applied over solid categorical colors in charts. They are especially helpful when there is only one visual distinction between data points, ensuring that all users can easily differentiate between chart elements.
```

Comment:
Is there a reason why pattern fills should always be black? To achieve 3:1 contrast, but I assume white would be the same...